.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ jobs:
             - run: npm run init
             - run: npm run lint
```

Comment:
Currently `lint` is defined as:

```
"lint": "prettier --write \"{src,scripts,functions,configs,reporting}/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
```

We probably don't want to be running `prettier`'s `--write` in CI.

Propose that we add change `scripts` to something like:

```
// ..snip..
    "lint": "npm-run-all eslint:src prettier:lint",
    "eslint:src": "eslint src --ext .js,.jsx,.ts,.tsx",
    "eslint:fix": "eslint --fix ?something like the above?"
    "prettier": "prettier --write \"{src,scripts,functions,configs,reporting}/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
    "prettier:lint": "prettier \"{src,scripts,functions,configs,reporting}/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
    "test": "react-scripts test",
// ..snip..
```

- https://www.npmjs.com/package/npm-run-all